ocean_provider/routes.py

```diff
@@ -439,25 +403,108 @@ def download():
             request, requests_session, url, download_url, content_type
         )
 
-    except InvalidSignatureError as e:
-        msg = f'Consumer signature failed verification: {e}'
-        logger.error(msg, exc_info=1)
-        return jsonify(error=msg), 401
+    except Exception as e:
+        logger.error(
+            f"Error: {e}. \n"
+            f"Payload was: documentId={data.get('did')}, "
+            f"consumerAddress={data.get('consumerAddress')},"
+            f"serviceId={data.get('serviceId')}"
+            f"serviceType={data.get('serviceType')}",
+            exc_info=1
+        )
+        return jsonify(error=str(e)), 500
+
+
+@services.route('/access-token', methods=['GET'])
+@validate(AccessTokenRequest)
+def access_token():
```

Comment:
since the function name will be visible in the exposed endpoints, and other functions use camel case, let's stick with that naming here.

Reply:
this is not the endpoint, it is just the function name, which is used in the exposed endpoints dict when invoking the root endpoint

Reply:
done ✅